inspire_matcher/core.py

```diff
@@ -241,3 +235,96 @@ def _compile_nested_prefix(query, record):
 
 def _get_common_path(paths):
     return '.'.join(os.path.commonprefix([path.split('.') for path in paths]))
+
+
+def _compile_authors_query(query, record):
+    parsed_name = ParsedName(record.get("full_name"))
+    nested_query = _return_empty_nested_query("authors")
+    if "inner_hits" in query:
+        nested_query['query']['nested']['inner_hits'] = query['inner_hits']
+    if len(parsed_name) == 1 and '.' not in parsed_name.first:
+        nested_query['query']['nested']['query']['bool']['must'].append(
+            _compile_exact_match_query("authors.last_name", parsed_name.last)
+        )
+        return nested_query
+
+    bool_must_query = list()
+    bool_must_query.append(_compile_exact_match_query("authors.last_name", parsed_name.last))
```

Comment:
```suggestion
    bool_must_query = [_compile_exact_match_query("authors.last_name", parsed_name.last)] 
```
This should also work right?

Reply:
Yes, thanks :)